### PR TITLE
[KO] Translation of Enum lesson

### PR DESCRIPTION
**Pull request by rijgndqw012**
Any feedback is welcome :)

### CODE REVIEW

File: ko/lessons/basics/enum.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+콜렉션 내의 `최대(max)` 값을 반환합니다:
+
+```elixir
+iex> Enum.max([5, 3, 0, -1])
+5
+```
+
+### reduce
+
+`reduce`를 이용하면, 콜렉션을 하나의 값으로 추려낼 수 있습니다. 이를 이용하려면, 선택사항으로 축적자를(예를 들면 `10`) 함수에 전달합니다; 만약 축적자가 제공되지 않는다면, 콜렉션의 첫번째 원소가 대신 그 역할을 합니다:
```
Comment (rijgndqw012) [inline ko/lessons/basics/enum.md]: reduce 함수 내에서 계산을 반복하면 반복할수록 그 값이 축적이 되기 때문에 축적자로 번역했습니다.

**Comment by doomspork** [comment]
@devleoper do you want to take a look at this, if you're got some time?

**Comment by devleoper** [comment]
Sorry for late reviewing. Looks great! :+1:

**Comment by doomspork** [comment]
No worries @devleoper, thank you for your help.

Thank you @rijgndqw012 for the translation!

**Pull request merged**